addons/mentions/addon.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Highlight mentions",
+  "description": "Highlight when you are mentioned (e.g. @username).",
+  "tags": ["community", "profiles", "comments", "forums"],
+  "versionAdded": "1.34.0",
+  "customCssVariables": [
+    {
+      "name": "button-text",
+      "value": {
+        "type": "textColor",
+        "black": "#000",
+        "white": "#fff",
+        "source": {
+          "type": "settingValue",
+          "settingId": "color"
+        }
+      }
+    }
+  ],
+  "userscripts": [
+    {
+      "matches": ["projects", "studios", "profiles", "forums"],
+      "url": "userscript.js"
+    }
+  ],
+  "userstyles": [
+    {
+      "matches": ["projects", "studios", "profiles", "forums"],
+      "url": "userstyle.css"
+    }
+  ],
+  "settings": [
+    {
+      "name": "Highlight color",
+      "id": "color",
+      "type": "color",
+      "default": "#855cd6"
```

Comment:
Should we change this color to a shade of yellow/orange by default? Should we include two presets so the user can quickly switch it between yellow/orange and purple? Should we remove the setting entirely?

Reply:
It might make sense to just have it be a native Scratch color for more easy compatibility with dark-www

Reply:
The orange used for the message count is customizable with `dark-www` now.